docs/guide/plugins.md

```diff
@@ -0,0 +1,202 @@
+    <!-- we can link to our product product -->
+    <nuxt-link
+      :to="getProductLink"
+      v-if="product"
+      class="cool-promotion-plugin__link"
+    >
+      <h3>{{ getName }} in special price! {{ getPrice | price }}</h3>
+    </nuxt-link>
+  </div>
+</template>
+<script>
+import { getProducts } from "@shopware-pwa/shopware-6-client";
+import {
+  getProductName,
+  getProductRegularPrice,
+  getProductUrl,
+} from "@shopware-pwa/helpers";
+
+export default {
+  name: "CoolPromotionPlugin",
+  data() {
+    return {
+      product: null,
+    };
+  },
+  async mounted() {
+    // let's get the cheapest product
+    const productsResult = await getProducts({
+      sort: {
+        field: "price",
+        desc: false,
+      },
+      pagination: {
+        limit: 1,
+      },
+    });
+    this.product = productsResult.data[0];
+  },
+  computed: {
+    getName() {
+      return getProductName({ product: this.product });
+    },
+    getProductLink() {
+      return getProductUrl(this.product);
+    },
+    getPrice() {
+      return getProductRegularPrice({ product: this.product });
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+.cool-promotion-plugin {
+  display: flex;
+  justify-content: center;
+  border: 1px solid green;
+
+  &__link {
+    padding: 20px 5px;
+    min-height: 30px;
+    margin-right: 2rem;
+  }
+}
+</style>
+```
+
+so we achieved this:
+
+![local-plugin-2](./../assets/local-plugin-2.png)
+
+### Nesting plugins
+
+Okay, but what if we have two or more plugins for our `top-header-after` slot?
+Always the newest installed plugin in shopware instance is on top. But local plugins are over them. The easiest way is to picture that. On shopware instance, we have our HelloCody plugin, which displays random quotes. It injects into `top-header-after` slot as well. So we overrode that.
+
+:::warning Always provide a slot for other plugins
+We should always provide slots for other plugins; otherwise, we're blocking slot only for our plugin.
+:::
+
+Okay, so how we can allow other plugins to display as well if we're in `top-header-after`? We're providing Vue's default slot in our component, as simple as that:
+
+```vue {11}
+<template>
+  <div class="cool-promotion-plugin">
+    <!-- we can link to our product product -->
+    <nuxt-link
+      :to="getProductLink"
+      v-if="product"
+      class="cool-promotion-plugin__link"
+    >
+      <h3>{{ getName }} in special price! {{ getPrice | price }}</h3>
+    </nuxt-link>
+    <slot />
+  </div>
+</template>
+```
+
+So this allows our installed plugin to show up and present itself.
+
+![local-plugin-3](./../assets/local-plugin-3.png)
+
+In this case, it shows a random plugin bud it could be as well another promotion plugin. You always decide where to show slot with another plugin if exists. There are a lot more use cases. Like social login button - you're doing plugin for Facebook, but others may do plugins for Google, GitHub etc.
```

Comment:
"but" instead of "bud"

Reply:
https://github.com/DivanteLtd/shopware-pwa/pull/695/commits/bf2f4561abada74ea64364c83ecc4ec8028de7d8